LinqToDBPostGisNetTopologySuite.Tests/AffineTransformationsTests.cs

```diff
@@ -0,0 +1,258 @@
+        }
+        [Test]
+        public void TestSTScaleXYZ()
+        {
+            using (var db = new PostGisTestDataConnection(TestDatabaseConnectionString))
+            {
+                const string ewkt = "LINESTRING(1 2 3, 1 1 1)";
+                db.TestGeometries.Value(g => g.Id, 1).Value(p => p.Geometry, () => GeometryInput.STGeomFromEWKT(ewkt)).Insert();
+                var result = db.TestGeometries
+                    .Where(g => g.Id == 1)
+                    .Select(g => g.Geometry.STScale(0.5, 0.75, 0.8))
+                    .Select(g => GeometryOutput.STAsEWKT(g))
+                    .Single();
+                Assert.AreEqual(result, "LINESTRING(0.5 1.5 2.4,0.5 0.75 0.8)");
+            }
+        }
+        [Test]
+        public void TestSTScaleXY()
+        {
+            using (var db = new PostGisTestDataConnection(TestDatabaseConnectionString))
+            {
+                const string ewkt = "LINESTRING(1 2 3, 1 1 1)";
+                db.TestGeometries.Value(g => g.Id, 1).Value(p => p.Geometry, () => GeometryInput.STGeomFromEWKT(ewkt)).Insert();
+                var result = db.TestGeometries
+                    .Where(g => g.Id == 1)
+                    .Select(g => g.Geometry.STScale(0.5, 0.75))
+                    .Select(g => GeometryOutput.STAsEWKT(g))
+                    .Single();
+                Assert.AreEqual(result, "LINESTRING(0.5 1.5 3,0.5 0.75 1)");
+            }
+        }
+        [Test]
+        public void TestSTScaleXYZM()
+        {
+            using (var db = new PostGisTestDataConnection(TestDatabaseConnectionString))
+            {
+                const string ewkt = "LINESTRING(1 2 3 4, 1 1 1 1)";
+                db.TestGeometries.Value(g => g.Id, 1).Value(p => p.Geometry, () => GeometryInput.STGeomFromEWKT(ewkt)).Insert();
+                var result = db.TestGeometries
+                    .Where(g => g.Id == 1)
+                    .Select(g => g.Geometry.STScale(GeometryConstructors.STMakePoint(0.5, 0.75, 2, -1)))
+                    .Select(g => GeometryOutput.STAsEWKT(g))
+                    .Single();
+                Assert.AreEqual(result, "LINESTRING(0.5 1.5 6 -4,0.5 0.75 2 -1)");
+            }
+        }
+        [Test]
+        public void TestSTScaleFalseOrigin()
+        {
+            using (var db = new PostGisTestDataConnection(TestDatabaseConnectionString))
+            {
+                const string ewkt = "LINESTRING(1 1, 2 2)";
+                db.TestGeometries.Value(g => g.Id, 1).Value(p => p.Geometry, () => GeometryInput.STGeomFromEWKT(ewkt)).Insert();
+                var result = db.TestGeometries
+                    .Where(g => g.Id == 1)
+                    .Select(g => g.Geometry.STScale(GeometryInput.STGeomFromEWKT("POINT(2 2)"), GeometryInput.STGeomFromEWKT("POINT(1 1)")))
+                    .Select(g => GeometryOutput.STAsEWKT(g))
+                    .Single();
+                Assert.AreEqual(result, "LINESTRING(1 1,3 3)");
```

Comment:
The Assert method signature is
`AreEqual(expected, actual)`
so it should look as
` Assert.AreEqual("LINESTRING(1 1,3 3)", result);`